src/FsAutoComplete.BackgroundServices/Program.fs

```diff
@@ -100,14 +101,31 @@ type BackgroundServiceServer(state: State, client: FsacClient) =
 
     let getListOfFilesForProjectChecking file =
         match state.FileCheckOptions.TryFind file with
-        | None -> None
+        | None ->
+            if file.EndsWith ".fsx" then
+                state.Files.TryFind file |> Option.map (fun st ->
+                    async {
+                        let targetFramework = NETFrameworkInfoProvider.latestInstalledNETVersion ()
+                        let! opts = fsxBinder.GetProjectOptionsFromScriptBy(targetFramework, file, st.Lines |> String.concat "\n")
+                        let sf = getFilesFromOpts opts
+
+                        return
+                            sf
+                            |> Array.skipWhile (fun n -> (Utils.normalizePath n) <> (Utils.normalizePath file))
+                            |> Array.map (fun n -> (Utils.normalizePath n))
+                            |> Array.toList
```

Comment:
I'll note to move that in DPW.
@Krzysztof-Cieslak  you need the list of sources files for a script, and?

Reply:
I need list of the files after current file in project. I guess in case of fsx files it doesn't really make sense, but I'm using same bit of code in other places too.